tests/js/app/testHistoryStore.js

```diff
@@ -161,4 +162,48 @@ describe('History Store', function () {
         expect(history_store.get_action_state('article-search-visible').unpack())
             .toBeFalsy();
     });
+
+    it('can share a history item', function () {
+        expect(history_store.can_share()).toBe(false);
+
+        history_store.set_current_item_from_props({
+            page_type: 'article',
+            model: Eknc.ArticleObjectModel.new_from_props({
+                title: 'Endless OS',
+                ekn_id: 'ekn://article1',
+            }),
+        });
+        expect(history_store.can_share()).toBe(false);
+
+        history_store.set_current_item_from_props({
+            page_type: 'article',
```

Comment:
there is a Pages "enum" we should use, but since this was already used like that in this code, consider this a nit, and if you want to fix it, just add an extra commit fixing all.